server/channels/app/custom_profile_attributes.go

```diff
@@ -76,12 +78,12 @@ func (a *App) CreateCPAField(field *model.PropertyField) (*model.PropertyField,
 		return nil, model.NewAppError("CreateCPAField", "app.custom_profile_attributes.cpa_group_id.app_error", nil, "", http.StatusInternalServerError).Wrap(err)
 	}
 
-	existingFields, appErr := a.ListCPAFields()
-	if appErr != nil {
-		return nil, appErr
+	fieldCount, err := a.Srv().propertyService.CountPropertyFieldsForGroup(groupID)
```

Comment:
So this change was not strictly necessary right now because we use FieldLimit to also limit the PerPage of CPAFields. But if we end up decorrelating the pagination size and the limit of CPA field allowed (like we wanna keep 20 fields per page, but increase the limit to 50), this would have become an issue.